lib/openscap_parser/subs.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'openscap_parser/sub'
+
+module OpenscapParser
+  module Subs
+    def self.included(base)
+      base.class_eval do
+        def sub
+          return unless sub_node
+          Sub.new(parsed_xml: sub_node)
+        end
+
+        def sub_node
```

Comment:
There can be 0 to n `sub` nodes according to the spec. IMO this should be pluralized